css/style.css

```diff
@@ -1071,3 +1356,1390 @@ ul {
+    width: 100%;
+    height: 100%;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    color: #6070ff;
+    font-weight: 700;
+    font-size: 1.25vw;
+    line-height: 112%;
+  }
+
+  #icon-button-button:disabled {
+    background: transparent;
+  }
+
+  #div-button {
+    width: 0;
+    left: 0%;
+    top: 0%;
+    border: none;
+  }
+
+  #menu {
+    width: 0%;
+    aspect-ratio: 1 / 1;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .vector {
+    width: 0;
+    border: none;
+    background: transparent;
+  }
+
+  #menu-2 {
+    position: absolute;
+    width: 17.986111%;
+    aspect-ratio: 259 / 40;
+    left: 72%;
+    top: 0.312195%;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+    padding: 0;
+    gap: 1.4vw;
+  }
+
+  #menu-2-button {
+    background: transparent;
+    border: none;
+    color: #344563;
+    font-weight: 600;
+    font-size: 1.2vw;
+    line-height: 100%;
+  }
+
+  #headline {
+    position: absolute;
+    width: 100%;
+    aspect-ratio: 1440 / 846;
+    left: 0%;
+    top: 1.385366%;
+    z-index: 2;
+    margin: 0;
+    border-bottom-left-radius: 10vw;
+  }
+
+  #div-primary-text-1 {
+    position: absolute;
+    width: 33%;
+    aspect-ratio: 920 / 122;
+    left: 18.055556%;
+    top: 36%;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: normal;
+  }
+
+  #div-primary-text-2 {
+    position: absolute;
+    width: 63.888889%;
+    aspect-ratio: 920 / 98;
+    left: 18.055556%;
+    top: 48.7%;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .primary-text-1 span {
+    display: none;
+  }
+
+  .primary-text-1::after {
+    position: absolute;
+    top: -20%;
+    height: 100%;
+    width: 100%;
+    font-weight: 700;
+    font-size: 3.4vw;
+    line-height: 4vw;
+    color: #172b4d;
+    align-items: center;
+    content: "I'm Alexandra Glad to see you!";
+  }
+
+  .primary-text-2 {
+    height: 100%;
+    width: 100%;
+    font-weight: 400;
+    font-size: 1.38vw;
+    line-height: 2vw;
+    text-align: left;
+    color: #344563;
+  }
+
+  #div-social-media {
+    position: absolute;
+    width: 8.125%;
+    aspect-ratio: 117 / 24;
+    left: 18.055556%;
+    top: 63%;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .social-media-p {
+    height: 100%;
+    width: 100%;
+    font-weight: 500;
+    font-size: 1.09vw;
+    line-height: 2vw;
+    text-align: left;
+    color: #7f8cff;
+    color: black;
+    margin: 0;
+  }
+
+  #div-social-media-icons {
+    position: absolute;
+    width: 11.805556%;
+    aspect-ratio: 170 / 20;
+    left: 18.055556%;
+    top: 66%;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-between;
+    border: 1px solid green;
+    padding: 0;
+  }
```

Comment:
You seem to redefine the same properties with the media query as was done earlier outside it. Note that the initial styles given to elements still apply. All you needed to do was to modify the properties that should change from what they were.

For example, check out the styling for the social media icon below (defined in the mobile version) and compare it with the highlighted code above.
![image](https://user-images.githubusercontent.com/20554583/194475370-cf2c66d0-490e-4f54-a0c5-e67803dc5090.png)

The position, display, flex-direction, align-items, justify-content properties needed not to be redefined since they're never change from what they were in the mobile version. Doing this will reduce the amount of CSS you'll have to write and ensure your code is DRYer.